extensions/auth/portability-auth-smugmug/src/main/java/org/datatransferproject/auth/smugmug/SmugMugOAuthConfig.java

```diff
@@ -16,18 +16,23 @@
 
 package org.datatransferproject.auth.smugmug;
```

Comment:
nit: Can we extract the duplicate string literals into private static fields?

Reply:
I made most of them static. I think only duplicate literal I didn't make static is the "Full" permission in the SmugMug config. I'm going to leave that one for now, since it matches the pattern used elsewhere (keys may be static, but values are not) and I think that makes it easier to read.